sdks/python/src/opik/api_objects/experiment/experiment.py

```diff
@@ -1,12 +1,16 @@
 import functools
 import logging
+import time
```

Comment:
THe changes in this module can be reverted, experiment items improvements are handled in another PR.